doc-content/enterprise-only/openshift/rhpam711-authoring-ref.adoc

```diff
@@ -63,7 +63,7 @@ https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/
 |`BUSINESS_CENTRAL_CPU_REQUEST` | -- | Business Central Container CPU Request. | 1500m | True
 |`BUSINESS_CENTRAL_MEMORY_REQUEST` | -- | Business Central Container Memory Request. | 3Gi | True
 |`KIE_SERVER_MEMORY_LIMIT` | -- | KIE Server Container memory limit. | 2Gi | True
-|`KIE_SERVER_MEMORY_REQUEST` | -- | KIE Server Container memory Request. | 1.5Gi | True
+|`KIE_SERVER_MEMORY_REQUEST` | -- | KIE Server Container memory Request. | 1536Mi | True
```

Comment:
[technical] Most size values are stated in `Gi`, is there a reason for changing this to `1536Mi`?